### ISSUE TITLE
Few QoL addon mod suggestions...

### ISSUE THREAD

**Issue by ByteTemplar** [adam9899/Issues#131]
Now that we have bees in, I would suggest few JEI addon mods to make life bit easier:

https://www.curseforge.com/minecraft/mc-mods/beebetteratbees
_This mod adds a more clear bee-breeding guidance in form of breeding trees to NEI (1.7.10)/ JEI (1.12.2) to add a little bit of support to help you breed the bees you want to have._

https://www.curseforge.com/minecraft/mc-mods/gendustry-jei-addon
_Adds recipes from Gendustry's machines to JEI. Supports all machines from Gendustry (Except Industrial Apiary)._

For Agricraft,  just to see stats in Waila, when upgrading seeds:
https://www.curseforge.com/minecraft/mc-mods/wasaila

Just a suggestions, but I think having those would make things bit better to utilize some mods...
Ive been using em for while and havent had any problems either.

**Comment by majora2007** [comment]
Yes please on beebetteratbees. I know a lot of people don't like bees, but personally it's a very unique and fun way to play MC. beebetteratbees would definitely help navigate such a complicated mod.
